zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/gun/shoot/handler/DamageShotHandler.java

```diff
@@ -22,6 +23,7 @@
 /**
  * A {@link ShotHandler} that deals damage to targets.
  */
+@Model("zombies.gun.shot_handler.damage")
 public class DamageShotHandler implements ShotHandler {
```

Comment:
this is an example of an element that can have @Cache(true) to actually force caching - you only need one instance of this per data